lib/Sapi.php

```diff
@@ -67,7 +67,21 @@ static function sendResponse(ResponseInterface $response) {
             }
 
         }
-        file_put_contents('php://output', $response->getBody());
+
+        $body = $response->getBody();
+        if(is_null($body)) return;
+
+        $contentLength = $response->getHeader('Content-Length');
```

Comment:
found it, its actually a whole series of blog posts, because firefox need to revert the patches serveral times

http://daniel.haxx.se/blog/2014/06/16/firefox-and-partial-content/
http://daniel.haxx.se/blog/2014/10/26/stricter-http-1-1-framing-good-bye/
http://daniel.haxx.se/blog/2015/03/02/more-http-framing-attempts/

But it seems they have issues because of servers handling content length differently, which is not the case for use here. maybe its a interessting read nevertheless ;)